packages/graphql-serve/src/runtime.ts

```diff
@@ -25,19 +28,45 @@ export const createMongoDBClient = async (): Promise<MongoClient> => {
   return client
 }
 
+export function getConflictMapForModels(model: GraphQLSchema, conflictResolutionStrategy: ConflictResolutionStrategy): DataSyncModelConfigMap {
+  const map: DataSyncModelConfigMap = {};
+
+  const dataSyncModelTypes = filterModelTypes(model).filter((type: GraphQLObjectType) => {
+    return !!parseMetadata('datasync', type);
+  })
+
+  dataSyncModelTypes.forEach((type: GraphQLObjectType) => {
+    map[type.name] = {
+      enabled: true,
+      deltaTTL: 172800,
+      conflictResolution: conflictResolutionStrategy
+    }
+  })
```

Comment:
I suppose we need this because we do not have a way to supply default config?

Reply:
Not saying we should introduce it but more a question.

Reply:
So we do nto have any way to supply default config? 
How this even works when I provide empty object?

This is must have!

Reply:
> So we do nto have any way to supply default config?
How this even works when I provide empty object?

Default value for conflictResolution? Built into plugin

Do you mean default value for deltaTTL?

Reply:
> I suppose we need this because we do not have a way to supply default config?

@machi1990 Do you mean default conflict config itself or default deltaTTL?

Conflict config is used to `enable` conflicts as well, so I think maybe putting a default conflict config in the plugin would mean enabling conflicts for all models, do you mean something like a global config?

Reply:
Let's discuss here https://github.com/aerogear/graphback/issues/1902